app/javascript/packs/App.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Body from './Body'
+import MyHeader from './MyHeader'
+
+class App extends React.Component {
+
+    constructor() {
+        super();
+        this.state = {
+            cuisines: []
+
+        };
+    }
+
+    componentWillMount() {
+        fetch('http://localhost:3000/cuisines.json')
+            .then(response => response.json())
+            .then((cuisines) => this.setState({cuisines: cuisines}));
+    }
+
+    render() {
+        console.log(this.props.cuisines)
+        return (
```

Comment:
Use [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) for better readability

```js
const { cuisines }  = this.state;
```